www/views/lights.html

```diff
@@ -80,7 +80,25 @@
 								<button id="speedUp" onclick="appLampSpeedUp()">Speed Up</button><button id="speedDown" onclick="appLampSpeedDown()">Speed Down</button><br />
 								<button id="discoUp" onclick="appLampDiscoMode()">Disco Mode</button><button id="speedUpLong" onclick="appLampSpeedUpLong()">Exclude Lamp</button><br />
 							</div>
+							<div id="optionsRGBW" style="display:none;">
```

Comment:
Will this only show up for LimitedLed lights ? Or also for ZWave RGB(W)(WW) lights ? (hope only limitedled)